libs/agno/agno/agent/agent.py

```diff
@@ -81,6 +81,7 @@ class Agent:
     session_name: Optional[str] = None
     # Session state (stored in the database to persist across runs)
     session_state: Optional[Dict[str, Any]] = None
+    search_previous_sessions_history: bool = False
```

Comment:
Thinking DX maybe something on the lines of `access_to_previous_messages` is more clear? 

"sessions_history" makes me think about the session object. Not sure, your call!

Reply:
It is basically the session object it is searching. The session object is basically a set of runs, which is really just a set of messages and responses